waisntechtools/waisntechtools/settings.py

```diff
@@ -116,17 +111,29 @@
 
 USE_TZ = True
 
-
 # Static files (CSS, JavaScript, Images)
 # https://docs.djangoproject.com/en/2.1/howto/static-files/
 
 STATIC_URL = '/static/'
 
 # BEGIN Auth0
+WAISN_AUTH_DISABLED = environ.get('WAISN_AUTH_DISABLED', False)
+
+
+def environ_var(key):
+    """Used to allow tests to run without having to set the environment variables. If this gets unwieldy, we should
```

Comment:
A little confused about what's going on.  Just to clarify about these settings...

- By default, Auth0 is enabled and we are in a 'production setting'
- When running test `@override_settings(WAISN_AUTH_DISABLED=True, DEBUG=True)` is enacted on test views
- If running test, environment variables become negated...? (Why bother doing this if environment variables are already set and WAISN_AUTH_DISABLED = True? Doesn't WAISN_AUTH_DISABLED already do this?)
- With these factors should `DEBUG=False` in settings.py ?

Reply:
What do you mean by they become negated? By default, `WAISN_AUTH_DISABLED` is set to `False`. Both `WAISN_AUTH_DISABLED` and `DEBUG` need to both be `True` for authentication to be turned off.

In the future, I was thinking of having:
* `settings-local.py` - settings imported when running locally for development
* `settings-prod.py` - settings used by the production stack
* `settings-common.py` - settings that are the same for both local and prod.

I'm thinking that when we do that, we'll have `DEBUG=False` in the `settings-prod.py` file.

Reply:
It looks like I was confused about WAISN_AUTH_DISABLED and DEBUG being redundant by being set twice and with the same value but somehow overlooked Line 120 ` WAISN_AUTH_DISABLED = environ.get('WAISN_AUTH_DISABLED', False)` setting the variable to False instead of True.

Explanation of the DEBUG variable makes sense.